source/2020-08-25-accessibility-working-group-update.md

```diff
@@ -0,0 +1,76 @@
+
+This addon took inspiration from the pre-RFC for the (guid-for) helper, which resulted in [RFC 659, which proposes a new built-in template helper](https://github.com/emberjs/rfcs/pull/659), authored by [Steve Szczecina](https://github.com/steveszc). 
+
+[Rajasegar Chandran](https://github.com/rajasegar) and [Jen Weber](https://github.com/jenweber) updated the [Official Ember Guides](https://guides.emberjs.com/release/components/built-in-components/#toc_ways-to-associate-labels-and-inputs) with more explicit information for how to associate `<label>` and `<input>` elements. [Melanie Sumner](https://github.com/melsumner) also authored [ember-component-patterns](https://emberjs-1.gitbook.io/ember-component-patterns/form-components/input) that reflect best practices for `<input>` fields.
+
+Finally, support has been added to the [Ember Template Lint addon](https://github.com/ember-template-lint/ember-template-lint) with rules that require `<input>` elements to have [valid labels](https://github.com/ember-template-lint/ember-template-lint/pull/1105) and that there are [no duplicate labels](https://github.com/ember-template-lint/ember-template-lint/pull/1439). Additional work is now being done to check that there are [no duplicate ids](https://github.com/ember-template-lint/ember-template-lint/pull/1187) as well as [valid use of form groups](https://github.com/ember-template-lint/ember-template-lint/pull/1367).
+
+### Page Title Support
+
+In our [April 8th meeting](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-04/april-08.md), the team discussed how we thought we should approach page title support. More discussion throughout the month to follow led us to support the template-driven support for page titles in Ember apps. [Benjamin Jegard](https://github.com/kamikillerto), [Melanie Sumner](https://github.com/melsumner) and [Ricardo Mendes](https://github.com/locks) authored [RFC 645](https://github.com/emberjs/rfcs/pull/645), which proposes that the Ember Page Title addon be added to the blueprints for new Ember apps. After Framework Core Team discussion, it was decided that the addon needed some work before the RFC could be accepted; that work was outlined [in an issue](https://github.com/adopted-ember-addons/ember-page-title/issues/167) and is [currently underway](https://github.com/adopted-ember-addons/ember-page-title/pull/168), thanks to community member [Raido Kuli](https://github.com/raido)!
+
+## Planning and Design Stage
+
+After discussion and some analysis, it was determined that some issues required the attention of the [Framework Core Team](https://emberjs.com/teams/) to resolve. 
+
+### Support for `...attributes`
+
+As outlined in [Ember Issue 19026](https://github.com/emberjs/ember.js/issues/19026), `...attributes` does not have a guaranteed merge order, which is problematic for five ARIA attributes that can have multiple values in the form of an ID reference list and the order of these values matters.
```

Comment:
```suggestion
As outlined in [Ember Issue 19026](https://github.com/emberjs/ember.js/issues/19026), `...attributes` does not have a guaranteed merge order. This is problematic for 5 ARIA attributes because they can have multiple values in the form of an ID reference list and the order of these values matters.
```